.changeset/khaki-pets-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-documentation': minor
+---
+
+Added a documentation page for the angular pattern `custom-select` which is bades on the ng-bootstrap `dropdown` component.
```

Comment:
```suggestion
Added a documentation page for the angular `custom-select` component based on the ng-bootstrap `dropdown` component.
```
Nitpick: I would not say that it is a pattern. It's rather a component without encapsulation.